lib/vty.h

```diff
@@ -375,10 +375,12 @@ extern bool vty_set_include(struct vty *vty, const char *regexp);
 extern int vty_json(struct vty *vty, struct json_object *json);
 extern int vty_json_no_pretty(struct vty *vty, struct json_object *json);
 extern void vty_json_empty(struct vty *vty, struct json_object *json);
+#ifdef VTYSH
 /* post fd to be passed to the vtysh client
  * fd is owned by the VTY code after this and will be closed when done
  */
 extern void vty_pass_fd(struct vty *vty, int fd);
```

Comment:
why don't you `ifdef` the lib/vty.c:vty_pass_fd() too ?

Reply:
Where?